tests/testToDoList.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { test, expect, chromium } from '@playwright/test';
+
+import {toDoListPageObject} from "../page_object/toDoListPageObject";
+import * as toDoListData from "../data/toDoListData.json"
+
+test.beforeEach(async ({ page }) => {
+  await toDoListPageObject.navigate(page)
+});
+
+test('has title', async ({ page }) => {
+  await toDoListPageObject.checkPageTitle(page)
+});
+
+test('add to do item', async ({page}) => {
+  await toDoListPageObject.fillAddTodoItem(page, toDoListData.taskToAdd)
+  await toDoListPageObject.enterAddTodoItem(page)
+  await toDoListPageObject.checkIfTaskWasAdded(page, toDoListData.taskToAdd)
+})
+
+test('completed to do item', async ({page}) => {
+
+  await toDoListPageObject.fillAddTodoItem(page, toDoListData.taskToComplete)
+  await toDoListPageObject.enterAddTodoItem(page)
+  await toDoListPageObject.checkIfTaskWasAdded(page, toDoListData.taskToComplete)
+  await toDoListPageObject.clickOnCompletedTask(page, toDoListData.taskToComplete)
+  await toDoListPageObject.chceckIfTaskIsCompleted(page, toDoListData.taskToComplete)
+})
+
+test('count tasks on list', async ({page}) => {
+  await toDoListPageObject.checkHowManyElementsOnList(page, 3)
```

Comment:
1. get count and assign to variable.
2. add new item
3. assert variable + 1 with getCount
and please don't use hardcoded int.